lib/galaxy/util/__init__.py

```diff
@@ -721,7 +721,7 @@ def __init__(self, params, sanitize=True):
                     key not in self.NEVER_SANITIZE and
                     True not in [key.endswith("|%s" % nonsanitize_parameter) for
                                  nonsanitize_parameter in self.NEVER_SANITIZE]):
```

Comment:
Yeah, I've read the pep and see the examples.  I still think the vertical alignment is valid (per the pep), easier to grasp the components of the conditional expression at a glance, and reasonable because there's still a visual break prior to the child block.